Fix start, stop, and restart

vbotka opened the pull request:
* Remove unused vars rc,out,_msg
* Add {out} to success msg instead of {rc}
* Stop jail if not down
* Jail start, stop, and restart functions always return
  changed=True. Start if not up. Stop if not down. Restart without
  checking state (iocage util will start jail if down).

vbotka commented:
All tests passed. See the previous PR.

```
ANSIBLE_SHOW_CUSTOM_STATS=true ansible-playbook iocage_test.yml -M . -e my_hosts=test_23

CUSTOM STATS: **********************************************************************************
	test_23:   a1: Aug 31 00:57:06  a2: Aug 31 01:04:42  ok: 47
```

xmj commented:
Changes lgtm :+1: but it now conflicts with what I've merved from #35 - could you please your changes on master branch?

You don't need to open a new pull request for this, just do `git checkout master ;  git pull origin master ; git checkout fix-start-stop ; git rebase -i master ; git push --force origin fix-start-stop` or something along these lines

vbotka commented:
> Changes lgtm  but it now conflicts with what I've merved from #35 - could you please your changes on master branch?

Actually, I was able to resolve it online here. Thank you!

xmj commented:
The reason I wanted you to rebase it is to not clutter the history with merges going from `master` into a `feature branch`, that will then get merged back into `master` again ;-)

xmj commented:
Git book reference -- https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

vbotka commented:
> The reason I wanted you to rebase it is to not clutter the history with merges going from `master` into a `feature branch`, that will then get merged back into `master` again ;-)

Done. (Thank you for the link!)

vbotka committed:
Fix start, stop, and restart

* Remove unused vars rc,out,_msg
* Add {out} to success msg instead of {rc}
* Stop jail if not down
* Test restarted state if not check_mode
* Jail start, stop, and restart functions always return
  changed=True. Start if not up. Stop if not down. Restart without
  checking state (iocage util will start jail if down).
